aea/configurations/base.py

```diff
@@ -1373,14 +1374,16 @@ def package_dependencies(self) -> Set[ComponentId]:
     def private_key_paths_dict(self) -> Dict[str, str]:
         """Get dictionary version of private key paths."""
         return {  # pylint: disable=unnecessary-comprehension
-            key: path for key, path in self.private_key_paths.read_all()
+            key: str(Path(path).as_posix())
+            for key, path in self.private_key_paths.read_all()
         }
 
     @property
     def connection_private_key_paths_dict(self) -> Dict[str, str]:
         """Get dictionary version of connection private key paths."""
         return {  # pylint: disable=unnecessary-comprehension
-            key: path for key, path in self.connection_private_key_paths.read_all()
+            key: str(Path(path).as_posix())
```

Comment:
I think you can extract `str(Path(path).as_posix())` into a utility function, say `as_posix_string`?